sdk/eventgrid/azure-eventgrid/README.md

```diff
@@ -1,21 +1,122 @@
-# Microsoft Azure SDK for Python
+# Azure EventGrid client library for Python
 
-This is the Microsoft Azure Event Grid Client Library.
-This package has been tested with Python 2.7, 3.5, 3.6, 3.7 and 3.8.
-For a more complete view of Azure libraries, see the [azure sdk python release](https://aka.ms/azsdk/python/all).
+Azure Event Grid is a fully-managed intelligent event routing service that allows for uniform event consumption using a publish-subscribe model.
 
+[Source code][python-eg-src] | [Package (PyPI)][python-eg-pypi] | [API reference documentation][python-eg-ref-docs]| [Product documentation][python-eg-product-docs] | [Samples][python-eg-samples]
```

Comment:
```suggestion
[Source code][python-eg-src] | [Package (PyPI)][python-eg-pypi] | [API reference documentation][python-eg-ref-docs]| [Product documentation][python-eg-product-docs] | [Samples][python-eg-samples] | [Changelog][python-eg-changelog]
```

(Just adding a changelog entry to the top navbar, this was feedback we got in SB, obviously will need to add the corrosponding link ref at the bottom)

Reply:
added